dbc2val/test/test_j1939/test_j1939.py

```diff
@@ -1,72 +0,0 @@
-#!/usr/bin/python3
-
-########################################################################
-# Copyright (c) 2023 Contributors to the Eclipse Foundation
```

Comment:
Intentional to remove this file? - it was just added, right?

Reply:
it's the latter :-)